def print_error(message):
    print(message)
    exit(-1)
